code/lib/store/src/types.ts

```diff
@@ -106,6 +106,14 @@ export type BoundStory<TFramework extends AnyFramework = AnyFramework> = Story<T
   storyFn: PartialStoryFn<TFramework>;
 };
 
+export declare type PlayContext<TFramework extends AnyFramework = AnyFramework> =
+  StoryContext<TFramework> & {
+    step: (
+      name: string,
+      play: (context: PlayContext<TFramework>) => MaybePromise<void>
+    ) => MaybePromise<void>;
+  };
```

Comment:
This is defined in `@storybook/csf` already: https://github.com/ComponentDriven/csf/blob/f3e3044802f41c4885e8f564ed9ec140f6f845a6/src/story.ts#L116-L122

Reply:
Turns out we don't need it at all.